eng/scripts/Language-Settings.ps1

```diff
@@ -257,3 +257,9 @@ function GetExistingPackageVersions ($PackageName, $GroupId = $null)
     return $null
   }
 }
+
+# Turn the package name start with `@azure/identity` to "identity".
+function Normalize-javascript-Package-name ($PackageName) 
+{
+  return $PackageName -replace "@azure/" , ""
```

Comment:
As I called out in your other PR lets try adding this as another property to the package properties so we don't have yet another small simple function that all languages have to implement.